src/etc/apple-darwin.supp

```diff
@@ -1,75 +0,0 @@
-{
```

Comment:
how come these files are no longer needed? where do valgrind suppressions come from these days? I grepped around and couldn't find it.